experimental/examples/prometheus/package.json

```diff
@@ -10,7 +10,8 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@opentelemetry/api": "^1.0.2",
-    "@opentelemetry/exporter-prometheus": "0.32.0",
-    "@opentelemetry/sdk-metrics": "0.32.0"
+    "@opentelemetry/exporter-prometheus": "0.33.0",
+    "@opentelemetry/sdk-metrics": "0.33.0",
+    "@opentelemetry/resources": "1.7.0"
```

Comment:
resource added to the example as a dependency but no code change?

Reply:
Ah that was intended for the actual exporter package, I keep adding this to the wrong package it seems. Thanks for catching this. Fixed in [993199c](https://github.com/open-telemetry/opentelemetry-js/pull/3300/commits/993199c358f7dd578eb00b2ce6bc7f25f6db63cd) :slightly_smiling_face: